Add FSI and Healthcare themes to archetype and update Easel and WKND themes

pankaj-parashar opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
Changes for cloud service only :
1. Including FSI and healthcare themes
2. Updating easel and wknd themes

## Description

<!--- Describe your changes in detail -->

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

pankaj-parashar committed:
FORMS-10696 - Add FSI and Healthcare theme to archetype

pankaj-parashar committed:
FORMS-10696 - Add FSI and Healthcare theme to archetype

Adding jcr:createdBy property for tracking purpose

pankaj-parashar committed:
FORMS-10696 - Add FSI and Healthcare theme to archetype

Updating thumbnail

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

    Removing jcr:createdBy property to fix violation

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Updating theme.zip and clientlibs for all the themes post all issue fixes

rismehta commented on src/main/archetype/ui.apps/src/main/content/jcr_root/apps/fd/af/theme-clientlibs/adaptiveform.theme.healthcare/.content.xml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[adaptiveform.theme.healthcare]"
```
Please use customer appId as name in theme category

pankaj-parashar replied:
`${appId}.adaptiveform.theme.healthcare` is new category name for healthcare theme. Similar pattern for other themes.

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Adding appId in theme paths, title and category name.
Adding entries for new templates to post genrate groovy script

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Changing theme name patterns for cloud service - using pattern ${appId} - <themeName>
Excluding theme clientlibs for 65 now.

pankaj-parashar committed:
Merge branch 'develop' into ootb-themes

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Excluding theme filters for 65

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Restructuring theme folders to include appId in theme folder names

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Adding merge mode in filter for themes folder

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Commenting out filter entry for theme client lib in apps.structure package for 6.5

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Fixing if else condition syntax issue

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Removing 65 theme inclusion commented entries to fix build issues.

rismehta commented on src/main/archetype/ui.apps/src/main/content/jcr_root/apps/fd/af/theme-clientlibs/__appId__.adaptiveform.theme.healthcare/css/theme.css.map:
```diff
@@ -0,0 +1 @@
+{"mappings":"AE8mBA,2XC7mBI,2IACE,4IACA,yIACA,kJACA,6ICmBN,wDAOE,2DAMA,2BAMA,2C,iDAqCF,mIArBI,2fAGE,+DAUF,kGAVE,wFADA,0FAYA,2GAAA,iGAZA,mGADA,iG,kG,wFAEA,yFADA,0FADA,0G,2G,iGADF,kG,mGAGE,sG,sG,kG,wFAAA,8FADA,8FADA,0FADF,+GAcE,+GAAA,2GAAA,iGAZA,uGADA,uGADF,mG,iGAaA,iGAAA,iGAAA,kGAVE,wFADA,yFADA,yFADF,yF,0FAGE,0GAWA,0GAAA,0GAAA,2G,iGAXA,kGADA,kGADA,kGADF,mG,iG,iG,iG,iGAAA,kGACE,wFADF,yF,yFAGE,yFADA ...
```
Can you remove the source map files ?

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Adding README for ui.apps package providing information about aem forms core component theme.

pankaj-parashar committed:
Add FSI and Healthcare themes to archetype and update Easel and WKND themes

Adding README for ui.apps package providing information about aem forms core component theme.

pankaj-parashar committed:
iAdd FSI and Healthcare themes to archetype and update Easel and WKND themes

Removing theme clientlibs for 65. Will add it back after 6.5.19.0
